README.md

```diff
@@ -7,18 +7,24 @@ The Lilt REST API enables programmatic access to the full-range of Lilt backend
   * Translation memory synchronization
 
 Requests and responses are in JSON format. The REST API only responds to HTTPS / SSL requests.
+
 ## Authentication
+
 Requests are authenticated via REST API key, which requires the Business plan.
 
 Requests are authenticated using [HTTP Basic Auth](https://en.wikipedia.org/wiki/Basic_access_authentication). Add your REST API key as both the `username` and `password`.
 
 For development, you may also pass the REST API key via the `key` query parameter. This is less secure than HTTP Basic Auth, and is not recommended for production use.
 
+## Quotas
+
+Our services have a general quota of 4000 requests per minute. Should you hit the maximum requests per minute, you will need to wait 60 seconds before you can send another request.
+
 
 This Python package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
 
 - API version: v2.0
-- Package version: 0.6.5
+- Package version: 1.0.0
```

Comment:
the automation script `openapi.yaml` will pass in the package version based on the tags. we should keep this package version the same until we bump the version to publish a new version of the package. this version number probably shows up elsewhere in the generated code/docs as well

Reply:
generated the code with the option `-p packageVersion=0.6.5` to fix this